Fantome/ModManagement/LeagueFileIndex.cs

```diff
@@ -77,6 +79,19 @@ public LeagueFileIndex(string leagueFolder)
             }
 
             Write();
+            //Temporary fix to load files
+            Application.Current.Shutdown();
+            Process process = null;
+            try
+            {
+                process = Process.GetProcessById(35076);
```

Comment:
The ID of a process is different every time you start it. Which makes this code totally useless as it will fail most of the time.

Reply:
I knew but idk how to get the pid for the opening fantome process